controllers/openstackmachine_controller.go

```diff
@@ -397,7 +397,7 @@ func (r *OpenStackMachineReconciler) reconcileLoadBalancerMember(logger logr.Log
 		return err
 	}
 
-	return loadbalancerService.ReconcileLoadBalancerMember(clusterName, machine, openStackMachine, openStackCluster, ip)
+	return loadbalancerService.ReconcileLoadBalancerMember(machine, openStackMachine, openStackCluster, clusterName, ip)
```

Comment:
I would also move `openStackCluster` to the first place

Reply:
fixed.